music21/converter/subConverters.py

```diff
@@ -952,7 +952,11 @@ def writeDataStream(self, fp, dataBytes: bytes):  # pragma: no cover
         return fp
 
     def write(self, obj, fmt, fp=None, subformats=None, **keywords):  # pragma: no cover
-        from music21.musicxml import m21ToXml
```

Comment:
Add `compress=False` to the signature so people know what is available here, I *think* -- it violates some subclassing paradigms but fits with what we have done in the rest of the system.